packages/api/portfolioDrafts/getPortfolioDraft.test.ts

```diff
@@ -0,0 +1,52 @@
+import { DynamoDBDocumentClient, GetCommand, GetCommandOutput } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { mockClient } from "aws-sdk-client-mock";
+import { handler, NO_PORTFOLIO_PATH_PARAM, NO_SUCH_PORTFOLIO } from "./getPortfolioDraft";
+import { DATABASE_ERROR } from "../utils/errors";
+import { ErrorStatusCode, SuccessStatusCode } from "../utils/response";
+
+const ddbMock = mockClient(DynamoDBDocumentClient);
+beforeEach(() => {
+  ddbMock.reset();
+});
+
+const validRequest: APIGatewayProxyEvent = {
+  pathParameters: { portfolioDraftId: "1234" },
+} as any;
+
+describe("Validation tests", () => {
+  it("should require path param", async () => {
+    const emptyRequest: APIGatewayProxyEvent = {} as any;
+    expect(await handler(emptyRequest)).toEqual(NO_PORTFOLIO_PATH_PARAM);
```

Comment:
Suggest using existing response from `errors.ts`
```suggestion
    expect(await handler(emptyRequest)).toEqual(PATH_VARIABLE_REQUIRED_BUT_MISSING);
```